Doc/library/math.rst

```diff
@@ -364,9 +364,8 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return *e* raised to the power *x*, where *e* = 2.718281... is the base
-   of natural logarithms.  This is usually more accurate than ``math.e ** x``
-   or ``pow(math.e, x)``.
+   Return the exponential function at *x*.  This is usually more accurate
```

Comment:
Again, the new description is ambiguous; I think the existing description is better.